### ISSUE TITLE
FR: CPU profiles

### ISSUE THREAD

**Issue by steadfasterX** [jsalatas/plasma-pstate#26]
it would be great to have customizable cpu profiles like shown in that gnome extension:

https://extensions.gnome.org/extension/945/cpu-power-manager/

**Comment by jsalatas** [comment]
Indeed! I was  also thinking for some time now about saved presets/profiles  :)

**Comment by mdedetrich** [comment]
A more general question, shouldn't this be part of KDE's central system settings rather than in the plasma widget? I think it makes more sense there.

**Comment by jsalatas** [comment]
> shouldn't this be part of KDE's central system settings rather than in the plasma widget? 

Yes. I'm currently working on that. See also https://github.com/jsalatas/plasma-pstate/issues/58
